Resources/ServerInfo/Guidebook/_RMC14/Law/RMCMarineLawArresting.xml

```diff
@@ -0,0 +1,59 @@
+  - Should the suspect be declared innocent of all accused crimes, the requesting officer may incur a Prevarication charge. The time for any crimes the suspect committed as a result of the arrest must still be served, however.
+  - Before the suspect is detained, the Officer may recind the order.
+  - In the case of [bold]officer ordered arrests[/bold] the [color=#8B0000][bold]Chief MP[/bold][/color] or [color=#008000][bold]Commander[/bold][/color] may choose to assign an NJP over the initial charge (where valid).
+
+  ## Arresting a Suspect
+  [italic]These arrests can be performed anywhere on the warship.[/italic]
+  - Take down the Suspect if they resist and securely restrain them.
+  - Move the Suspect to a brig cell.
+  - Continue to [bold]charging procedure[/bold].
+
+  ## Arresting MP
+  - The [color=#8B0000][bold]Chief MP[/bold][/color] can only be arrested if approved by the [color=#008000][bold]Commander[/bold][/color] or [color=#ffa500][bold]Provost Marshal Office[/bold][/color].
+  - Any [color=#ff000d][bold]MP[/bold][/color]s can only be arrested if approved by the [color=#008000][bold]Commander[/bold][/color] or [color=#8B0000][bold]Chief MP[/bold][/color]. The [color=#008000][bold]Commander[/bold][/color] has the final say.
+  - A [color=#90EE90][bold]Duty Officer[/bold][/color] is unable to authorise the arrest of an [color=#ff000d][bold]MP[/bold][/color] or the [color=#8B0000][bold]Chief MP[/bold][/color].
+  - Ensure additional charging of Military Police procedure is followed.
+
+  ## Actions Against the Commander
+  - [color=#ff000d][bold]MP[/bold][/color] must contact [color=#ffa500][bold]Provost Marshal Office[/bold][/color] before taking any law-related actions against the [color=#00CBFE][bold]Commanding Officer[/bold][/color].
+  - If there is no [color=#00CBFE][bold]Commanding Officer[/bold][/color], the [color=#008000][bold]Commander[/bold][/color] of the operation may be arrested, but [color=#ffa500][bold]Provost Marshal Office[/bold][/color] must be notified after the fact, through fax.
+
+
+  ## Field Arrests
+  [italic]These refer to arrests performed in the area of operations.[/italic]
+  - Field arrests are restricted to the FOB, dropships, and secure areas unless in pursuit.
+  - If a suspect flees outside the above areas or are witnessed by the [color=#ff000d][bold]MP[/bold][/color], they may be pursued.
+  - If there are hostiles in the vicinity, the [color=#ff000d][bold]MP[/bold][/color] must abort the arrest and move to a secure area.
+  - [color=#ff000d][bold]MP[/bold][/color]s may move between secure areas.
+  - [bold]Squad Leaders[/bold] may detain marines of their squad that violate the law, but must detain marines of their squad by orders of the [color=#8B0000][bold]Chief MP[/bold][/color], [color=#008000][bold]Commander[/bold][/color], or any commisioned officer. This may also apply to personnel that are deploying unauthorised.
```

Comment:
```suggestion
  - [bold]Squad Leaders[/bold] may detain marines of their squad that violate the law, but must detain marines of their squad by orders of the [color=#8B0000][bold]Chief MP[/bold][/color], [color=#008000][bold]Commander[/bold][/color], or any commissioned officer. This may also apply to personnel that are deploying unauthorized.
```
commi**ss**ioned and unathori**z**ed.